documentation/asciidoc/accessories/camera/camera_hardware.adoc

```diff
@@ -141,68 +131,12 @@ Then, please follow the relevant setup instructions either for xref:camera.adoc#
 | 2.9
 | 2.0
 | Depends on lens
-|===
-
-=== Software Features
-
-==== `libcamera` stack
```

Comment:
It looks like these tables have been deleted and not moved elsewhere - is that because they're now redundant?

Reply:
That's correct.  In the commit message I commented that they don't really belong in this type of documentation.  Perhaps in future, when we have a "product brief" type page, they can be resurrected there. They are mostly inaccurate as well :)